src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPage.tsx

```diff
@@ -33,38 +32,40 @@ const ActionDetailsPage = () => {
   const { colony } = useColonyContext();
   const { transactionHash, colonyName } = useParams<ActionDetailsPageParams>();
 
-  const action = mockActionData.find(
-    (item) => item.transactionHash === transactionHash,
-  );
-
-  const createdAt = action?.createdAt;
+  const { data, loading } = useGetColonyActionQuery({
```

Comment:
I imagine the queries / hooks here being wrapped up into a single `useGetColonyAction` hook, such as https://github.com/JoinColony/colonyCDapp/blob/ddfba48a30746c2617878456cfeead0a567d4905/src/hooks/useGetColonyAction.ts

Reply:
Cool, I think it makes sense for your PR to bring those changes rather than replicating them here